src/sentry/static/sentry/app/components/shortId.tsx

```diff
@@ -44,7 +45,7 @@ const StyledShortId = styled('div')`
   justify-content: flex-end;
 `;
 
-const StyledAutoSelectText = styled(AutoSelectText)<{
+const StyledAutoSelectText = styled(AutoSelectText, {shouldForwardProp: isPropValid})<{
```

Comment:
I think we need to forward if is valid prop and prop is not `avatar` (`isPropValid` is default for `shouldForwardProp`)